incubator/elasticsearch/templates/coordinating-deploy.yaml

```diff
@@ -1,34 +1,34 @@
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: "{{ template "client.fullname" . }}"
+  name: "{{ template "coordinating.fullname" . }}"
   labels:
     app: "{{ template "name" . }}"
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
Add a helper name `elasticsearch.chart` for this see https://github.com/bitnami/charts/blob/master/incubator/node/templates/_helpers.tpl#L29-L31